### PR TITLE
[Form] Fixed by_reference option with explicit cloning

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: reference/forms/types/options/by_reference.rst.inc [hunk 1]
```diff
@@ -31,7 +31,7 @@ Notice that ``setAuthor()`` is not called. The author is modified by reference.
 If you set ``by_reference`` to false, submitting looks like this::
 
     $article->setTitle('...');
-    $author = $article->getAuthor();
+    $author = (clone) $article->getAuthor();
```
Comment (wouterj) [inline reference/forms/types/options/by_reference.rst.inc]: Very minor, but `clone` isn't a type cast so please remove the `()` to avoid confusion.

**Comment by wouterj** [comment]
👍 
status: reviewed

File: reference/forms/types/options/by_reference.rst.inc (hunk 1 above)
Comment (HeahDude) [inline reference/forms/types/options/by_reference.rst.inc]: Of course, fixed, thanks!

**Comment by xabbuh** [comment]
Thank you @HeahDude.

**Pull request merged**